DataDictionaries/Epi25GGE.csv

```diff
@@ -72,7 +75,8 @@ eeg_findings_3_epicom,clinical,,notes,Other epileptiform comments,,,,,,,[eeg_fin
 eeg_findings_ppr_3,clinical,,dropdown,Type of photoparoxysmal response,"1, Generalized | 2, Occipital | 3, Other focal | 4, Non-epileptiform | 998, Unknown",,,,,,[eeg_findings_3] = '9',y,,,,,
 neuroimaging_performed,clinical,Neuroimaging,radio,Neuroimaging performed,"1, CT | 2, MRI | 997, Not done | 3, CT and MRI | 998, Unknown",,,,,,,,,,,,
 neuroimaging_findings_gge,clinical,,dropdown,Neuroimaging findings,"1, Normal | 2, Abnormal | 3, Non-specific abnormality, please specify | 996, Other, please specify | 998, Unknown",,,,,,[neuroimaging_performed]<997,,,,,,
-neuroimaging_specify,clinical,,notes,Neuroimaging findings comments,,,,,,,[neuroimaging_findings_gge] > '1' and [neuroimaging_findings_gge] < '997',,,,,,
+neuroimaging_findings,clinical,,dropdown,Neuroimaging findings - Focal <br>Stop gap solution,"1, Normal | 2, Nonspecific abnormality, please specify | 3, Hippocampal sclerosis | 100, Malformation: Focal Cortical Dysplasia | 101, Malformation: Heterotopia | 102, Malformation: Peri-ventricular nodular heterotopia | 103, Malformation: Polymicrogyria | 104, Malformation: Lissencephaly | 105, Malformation: Double Cortex | 106, Malformation: Hypothalamic Hamartoma | 107, Malformation: Hemimegaencephaly | 108, Malformation: Schizencephaly | 150, Benign Tumor: DNET | 151, Benign Tumor: Ganglioglioma | 152, Benign Tumor: unknown | 160, Traumatic Brain Injury | 170, Vascular and/or ischemic abnormalities: ischemic stroke | 171, Vascular and/or ischemic abnormalities: hypoxic ischemic injury | 172, Vascular and/or ischemic abnormalities: hemorrhage | 173, Vascular and/or ischemic abnormalities: cerebral angioma | 996, Other, please specify | 998, Unknown",,,,,,[neuroimaging_performed]<997 and [mixed_case]=1,,,,,,"TODO: Overhaul display logic and merge neuroimaging_findings between EE, GGE and Focal"
+neuroimaging_specify,clinical,,notes,Neuroimaging findings comments,,,,,,,[neuroimaging_findings_gge] > '1' and [neuroimaging_findings_gge] < '997',,,,,,TODO: Overhaul display logic

```

Comment:
Already addressed on master:
Field added in 9e7fd21d58611dfd06cf51839a924295edb128e0
Updated in 5b57b230d68b6105e5d69a79dbb2b075e863fd84